docs/chat/chat.yaml

```diff
@@ -48,4 +48,67 @@ paths:
               example:
                 status: 401
                 code: 'UNAUTHORIZED'
-                message: 'The requested URL requires user authorization.'
\ No newline at end of file
+                message: 'The requested URL requires user authorization.'
+  /chats/{id}/messages:
+    get:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Chats
+      summary: Find your messages in chat
+      description: Finds and returns an array with a list of all your messages related to specific chat.
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the chat that needs to be fetched messages from
+          type: string
```

Comment:
add fields `skip` and `limit` to parameters of query

Reply:
fixed